maintainers/scripts/update-python-libraries

```diff
@@ -169,7 +182,24 @@ def _get_latest_version_pypi(package, extension, current_version, target):
     return version, sha256
 
 
-def _get_latest_version_github(package, extension, current_version, target):
+def _get_latest_version_github(text, package, extension, current_version, target):
+    owner = _get_unique_value('owner', text)
+    repo = _get_unique_value('repo', text)
+    try:
+        # "Draft releases and prereleases are not returned by this endpoint"
+        # https://developer.github.com/v3/repos/releases/#get-the-latest-release
+        release = github.get_user(owner).get_repo(repo).get_latest_release()
+        tag = release.tag_name
+        # TODO: more fine-grained parsing, possibly using the info from the 'rev' attribute
+        if tag[0] == 'v':
+            version = tag[1:]
+        else:
+            version = tag
+        sha256 = nix_prefetch_github(owner=owner, repo=repo, rev=tag)['sha256']
```

Comment:
Why is `nix_prefetch_github` needed? The API does not offer sha256? Or is it because it does the choosing between zip and git?

Reply:
Isn't the hash for `fetchFromGitHub` the hash of the unpacked archive? To calculate it, I think I'd need to download and unpack it. That's what I use `nix_prefetch_github` for.

Reply:
When `fetchFromGitHub` picks an archive, it uses `fetchzip` and for that `nix-prefetch-url --unpack` can be used to determine the hash.

Reply:
I can use that if you prefer.

Reply:
No need for that. I was just hoping we could use the API instead of having to perform a download :)

Reply:
Note that the requests limits for downloading without token is quite low so you would likely to run into limits.